src/emulator/storage/rules/runtime.ts

```diff
@@ -313,6 +313,10 @@ function toExpressionValue(obj: any): ExpressionValue {
     return { string_value: obj };
   }
 
+  if (typeof obj == "boolean") {
+    return { bool_value: obj };
+  }
+
   if (typeof obj == "number") {
```

Comment:
Ideally we should have some tests for these branches and corner cases (e.g. `undefined`)